### DISCUSSION TITLE
sharing code w/ nodejs

### DISCUSSION THREAD

**Discussion by unicomp21** [RaananW/babylonjs-webpack-es6#39] [Q&A]
@RaananW if I want to share code between babylonjs and nodejs, what is the best way to do it in this project?

**Comment by unicomp21** [comment]
would lerna or yarn workspaces make sense?

**Reply by RaananW** [reply]
not sure why it would make sense to switch?
what is missing in npm? there are no workspaces here, it is a single project

**Reply by RaananW** [reply]
Oh, if you mean for the node version - first you can always use npm workspaces, which are great for small-sized mono-repos. second - I am not sure what the use case is. So, follow up questions will come :-)

**Comment by unicomp21** [comment]
typescript project references?

**Reply by RaananW** [reply]
what do you mean exactly?

**Comment by RaananW** [comment]
I'll open up an answer thread here.

What exactly are you looking for? a server based on node.js and a frontend based on babylon? or do you want to run babylon in a node environment as well?

**Comment by unicomp21** [comment]
Yes, a server based on node.js
Good point, you're right, due to usage of babylonjs math classes etc., I need babylon running in a node environment.

**Reply by RaananW** [reply]
Got it!

i'll add the option when I find the time :-) could be a nice experiment in npm workspaces and code sharing.

**Reply by unicomp21** [reply]
@RaananW thanks, appreciate you!  i'm definitely in your debt.
